services/brig/src/Brig/API/User.hs

```diff
@@ -262,14 +257,10 @@ createUser' viaScim new@NewUser {..} = do
           Just (inv, info, tid) -> (Nothing, Just (inv, info), Just tid)
           Nothing -> (Nothing, Nothing, Nothing)
     handleTeam (Just (NewTeamCreator t)) _ = do
-      -- invariant: viaScim /= CreateUserNoVerifyViaScim
-      -- FUTUREWORK: throw 400 in case of violation
       (Just t,Nothing,) <$> (Just . Id <$> liftIO nextRandom)
     handleTeam (Just (NewTeamMemberSSO tid)) _ = pure (Nothing, Nothing, Just tid)
-    handleTeam Nothing _ = do
-      -- invariant: viaScim /= CreateUserNoVerifyViaScim
-      -- FUTUREWORK: throw 400 in case of violation
-      return (Nothing, Nothing, Nothing)
+    handleTeam (Just (NewTeamMemberScimInvitation tid)) _ = pure (Nothing, Nothing, Just tid)
```

Comment:
not sure about this; note to self: double check!  in what situations is `handleTeam` called?